scripts/travis_build_and_test.sh

```diff
@@ -1,4 +1,3 @@
-
 #!/bin/bash
```

Comment:
@trws thanks! This was an obvious bonk on my part.  I'm going to crawl into a hole now....